src/style.css

```diff
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.searchForm {
+  margin-top: 20px;
+  position: relative;
+  box-sizing: border-box;
+}
+
+.searchLogo {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.smallLogo {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 15px;
+}
+
+.searchInput {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  width: 100%;
+  height: 70px;
```

Comment:
add box-sizing here, cause now the element's width is bigger than 70px, cause the border is includes
that's why the tests is not passed

Reply:
done